site/en/install/pip.md

```diff
@@ -70,6 +70,10 @@ step-by-step instructions.
     for CUDA in WSL.
 
     ```bash
+    # Windows WSL2 with Ubuntu 24.04 or later adheres to PEP 668, requiring users to manage their Python environments independently.
+    sudo apt install python3-venv
```

Comment:
Makes sense... I don't have a Windows box handy.. are we sure this install is necessary? isn't `venv` part of the base python installation?

Reply:
@gaikwadrahul8